exporter/datadogexporter/utils/trace_helpers.go

```diff
@@ -86,6 +91,10 @@ func NormalizeSpanName(tag string) string {
 		case unicode.IsDigit(c) || c == '.':
 			buf.WriteRune(c)
 			lastWasUnderscore = false
+		// '-' only creates issues for span operation names not service names
+		case c == '-' && isService:
+			buf.WriteRune(c)
+			lastWasUnderscore = false
```

Comment:
We should add a test for this

Reply:
added some more tests for the method generally, and for this particular case

Reply:
Weird, CodeCov is still complaining about this 🤔